### PR TITLE
Fix bugs and setup account system screen

### PR DESCRIPTION
**Pull request by evaldas-gr**
Added logger, created custom account screen, fixed blog, and user context bugs, and modified backend code for activity logging.
Added migration for database logging. We will need to do one more migration at minimum for auto-generated date field in the Activity model.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (kkersis) [inline .vscode/settings.json:1]: i think this file is unnecessary

Comment (evaldas-gr) [reply .vscode/settings.json:1]: visual studio 2022 bug on mac.

File: src/components/contexts/UserContext.tsx [hunk 2]
```diff
@@ -36,6 +38,18 @@ export const UserProvider = (props: { children: any }) => {
   const [token, setToken] = useState<string | null>(null);
   const [error, setError] = useState<string | null>(null);
   const [isLoading, setIsLoading] = useState(false);
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const defaultConnectionError = "Could not establish connection to server. Please try again!";
+
+  useEffect(() => {
+    console.log('jungiasi');
```
Comment (kkersis) [inline src/components/contexts/UserContext.tsx]: debugging leftovers

File: src/components/UserSettings/UserSettingsScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                        <div className="rightPanel col-12 col-lg-8 panelBox">
+                            <div className="header">
+                                <h1 className="bold header-title">Edit Profile</h1>
+                                <div className="header-choices">
+                                    <div className="choice" style={{borderBottom: '2px solid rgba(34,193,195,1)'}}>User info</div>
+                                    <div className="choice">Billing Information</div>
+                                </div>
+                            </div>
+                            <div style={{margin: '20px'}}>
+                            <form onSubmit={() => console.log('e')}>
```
Comment (kkersis) [inline src/components/UserSettings/UserSettingsScreen.tsx]: debugging leftovers

**Review by jasdom — APPROVED** [review]

**Review by kkersis — APPROVED** [review]

**Pull request merged**